Use Cases/SDPi/Temp from Repo/tf1-ch-c-use-cases.adoc

```diff
@@ -0,0 +1,88 @@
+= Appendix C – Device Point-of-care Interoperability (DPI) Use Cases
+
+
+NOTE:  This file integrates the use cases that are linked to the SDPi profiles to provide functional requirements, including any introductory sections for how use cases are managed in SDPi. +
```

Comment:
- Why is this a note?
- What is "this file"?

Is all this supposed to be TODO item?

Reply:
Note removed; a #TODO added to cover the rationale for the appendix.